src/utils/dns.js

```diff
@@ -53,53 +53,65 @@ const processAnswer = (type, answer) => {
 
                     // Get the tag, dropping any non alpha-numeric bytes per
                     //  https://tools.ietf.org/html/rfc6844#section-5.1
-                    const tag = words.splice(0, tagLength)
-                        .map(part => String.fromCharCode(parseInt(part, 16))).join('').trim()
+                    const tag = words
+                        .splice(0, tagLength)
+                        .map((part) => String.fromCharCode(parseInt(part, 16)))
+                        .join('')
+                        .trim()
                         .replace(/[^a-z0-9]/gi, '');
 
                     // Get the value
-                    const value = words.map(part => String.fromCharCode(parseInt(part, 16))).join('').trim();
+                    const value = words
+                        .map((part) => String.fromCharCode(parseInt(part, 16)))
+                        .join('')
+                        .trim();
 
                     // Combine and output
-                    entry.data = `${flags} ${tag} "${value}"`;
+                    entry.data = `${flags} ${tag} '${value}'`;
                     continue;
                 }
 
                 // Normal hex data
-                entry.data = words.map(part => String.fromCharCode(parseInt(part, 16))).join('').trim();
+                entry.data = words
+                    .map((part) => String.fromCharCode(parseInt(part, 16)))
+                    .join('')
+                    .trim();
             }
         }
     }
 
     return answer;
 };
 
-const performLookupJson = async (domain, type, endpoint) => {
+const performLookupJson = async (domain, type, endpoint, cdflag) => {
     // Build the query URL
     const query = new URL(endpoint.endpoint);
     query.searchParams.set('name', domain);
     query.searchParams.set('type', type.toLowerCase());
+    query.searchParams.set('cd', cdflag);
 
     // Make our request
     return fetch(query.href, {
         headers: {
             Accept: 'application/dns-json',
         },
-    }).then(res => res.json());
+    }).then((res) => res.json());
 };
 
 const randInt = (min, max) => Math.floor(Math.random() * (max - min + 1)) + min;
 
-const performLookupDns = async (domain, type, endpoint) => {
+const performLookupDns = async (domain, type, endpoint, cdflag) => {
     // Build the query packet
     const packet = encode({
         type: 'query',
         id: randInt(1, 65534),
-        flags: RECURSION_DESIRED,
-        questions: [ {
-            name: domain,
-            type,
-        } ],
+        flags: RECURSION_DESIRED | (cdflag && CHECKING_DISABLED),
```

Comment:
```suggestion
        flags: RECURSION_DESIRED | (cdflag ? CHECKING_DISABLED : 0),
```
Just a nitpick so we aren't relying on `false` type-casting to `0`